package com.leezp.sophix.exception;

public class Caclutor {
    public int caculator() {
        throw new RuntimeException("抛出异常了……");
    }
}
